dracut.sh

```diff
@@ -1954,19 +1954,28 @@ if [[ $no_kernel != yes ]]; then
 
     if [[ -n ${add_drivers// /} ]]; then
         # shellcheck disable=SC2086
-        hostonly='' instmods -c $add_drivers
+        if ! hostonly='' instmods -c $add_drivers; then
+            dfatal "instmods failed for add_drivers: $add_drivers"
```

Comment:
A new config parameter will have the same effect right? And users can modify to their needs.
If the logs are suppressed, users will never be aware that something is going wrong.

OTOH, agree with you on maintaining compatibility but it should be more flexible and configurable.

Reply:
Does this module has a test case? I can enhance it to cover this case, can you please point me where it's done currently?

Reply:
Thanks for your time and patience. Let's wait.

Reply:
> Perhaps this should be a warning and we should remove the exit to maintain compatibility.

`instmods()` already prints an error message, so adding another warning would be worthless.

IMHO I would not merge this PR as is, as it will break the current behavior.